src/iOS/toga_iOS/widgets/scrollcontainer.py

```diff
@@ -0,0 +1,104 @@
+from rubicon.objc import CGSizeMake
+from toga_iOS.libs import (
+    NSLayoutAttributeBottom,
+    NSLayoutAttributeHeight,
+    NSLayoutAttributeLeading,
+    NSLayoutAttributeTop,
+    NSLayoutAttributeTrailing,
+    NSLayoutAttributeWidth,
+    NSLayoutConstraint,
+    NSLayoutRelationEqual,
+    UIScrollView
+)
+from toga_iOS.window import (
+    iOSViewport,
+    UIColor
+)
+from travertino.size import at_least
+from .base import Widget
+
+
+class ScrollContainer(Widget):
+    __current_content = None
+    __scroll_height_constraint = None
+    __scroll_width_constraint = None
+    __vertical_enabled = False
+    __horizontal_enabled = False
```

Comment:
Why are these class-level properties?

Reply:
This is my mistake - I thought doing this will instantiate *instance* variables. I just learned this now after looking it up 👍 I'll make the change for this too.